redash/query_runner/athena.py

```diff
@@ -153,11 +154,20 @@ def get_schema(self, get_stats=False):
             raise Exception("Failed getting schema.")
 
         results = json_loads(results)
-        for row in results['rows']:
+        table_samples = {}
+
+        for i, row in enumerate(results['rows']):
             table_name = '{0}.{1}'.format(row['table_schema'], row['table_name'])
             if table_name not in schema:
-                schema[table_name] = {'name': table_name, 'columns': []}
+                schema[table_name] = {'name': table_name, 'columns': [], 'metadata': []}
+                table_samples[table_name] = self._get_table_sample(table_name)
```

Comment:
Some users might have hundreds if not thousands of tables. We should probably:

1. Make loading of table samples optional.
2. Separate it from loading the schema: first load the schema, then trigger sample data loading. Each table should be in its own task.

Also, not sure we should load samples on every schema refresh. Although this is an optimization we can perform later.

Reply:
I added a checkbox "Show Data Samples" in the data source for an admin to decide if these samples should be computed. They're also now computed in a separate task per table. 

In the next PR when the admin has controls per-table, we can make it possible to decide per table if samples are computed instead of per data source.